benchmark/framework/containerd_utils.go

```diff
@@ -205,7 +207,8 @@ func (proc *ContainerdProcess) CreateTask(
 func (proc *ContainerdProcess) RunContainerTaskForReadyLine(
 	ctx context.Context,
 	taskDetails *TaskDetails,
-	readyLine string) (func(), error) {
+	readyLine string,
+	parseFileAccessPatterns bool) (func(), error) {
```

Comment:
Maybe this was in another PR that I missed, but are we always emitting these logs but only parsing them if this flag is enabled? Do we need to do something to ensure that the logs are enabled when we try to parse them?

Reply:
We are not always emitting these logs. You need to run soci snapshotter in debug mode and also `log_fuse_operations` should be set to true in config.toml.